apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
```

Comment:
@lennym I started calling it `date-component` and searched for it in https://github.com/UKHomeOfficeForms until I remembered we called it `hof-component-date`.  I kept it `component-date` just for consistency.  I can change it if you like but it's a little confusing for a novice that in one place we are saying `date-component` and on the repo we say `hof-component-date`.  However, I do prefer the name date-component

Also, I do sometimes think of you as Yoda